uportal-war/src/main/java/org/apereo/portal/layout/dlm/remoting/UpdatePreferencesServlet.java

```diff
@@ -231,6 +232,65 @@ public ModelAndView removeElement(HttpServletRequest request, HttpServletRespons
     }
 
     /**
+     * Remove the first element with the provided fname from the layout.
+     *
+     * @param request
+     * @param response
+     * @param fname
+     * @return
+     * @throws IOException
```

Comment:
Could the autogenerated Javadocs lines 237-241, be removed or updated to include a description?